app/components/nav-rail.tsx

```
@@ -6,76 +6,100 @@ import NavRailPinnedItem from './nav-rail-pinned-item';
         items: {
             [key: string]: NavItemData
         },
+        pinned?: {
+            [key: string]: NavItemData
+        },
         selected: number
     }
 ) {
-    const [selected, setSelected] = useState(props.selected);
-    const [extended, setExtended] = useState(useMediaQuery('(min-width: 768px)'));
+    let sum_ = Array.from(Object.keys(props.items)).length + (props.pinned === undefined ? 0 : Array.from(Object.keys(props.pinned!)).length);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)